docs/source/quark_reports.rst

```diff
@@ -0,0 +1,129 @@
+++++++++++++++++++++++++++++++++++++++++
+Quark Report
+++++++++++++++++++++++++++++++++++++++++
+
+Easy to Use and Reading Friendly Report
```

Comment:
How about "reader-friendly report" or "intuitive report", instead of "Reading Friendly Report"

Reply:
Thanks for pointing out the typo! Will correct it.

Reply:
I've change it to "Intuitive Report". Thanks.